frontend/fingertips-frontend/components/pages/home/home.test.tsx

```diff
@@ -77,3 +77,35 @@ it('should focus on the input boxes when there is a validation error', async ()
   });
   expect(scrollMock).toBeCalledTimes(1);
 });
+
+describe('contents items should link to appropriate headings', () => {
+  interface TestData {
+    linkText: string;
+    headingText: string;
+  }
+  test.each<TestData>([
```

Comment:
We've used `it` instead of `test` in the rest of this file (and the rest of the app generally), so should stick with it here:
```suggestion
  it.each<TestData>([
```